src/plugins/vis_type_timeseries/public/application/components/aggs/positive_rate.js

```diff
@@ -167,7 +167,7 @@ export const PositiveRateAgg = (props) => {
         </EuiFlexItem>
       </EuiFlexGroup>
       <EuiSpacer size="s" />
-      <EuiText size="xs" color="subdued">
+      <EuiText size="s" color="subdued">
```

Comment:
Do you have a screenshot of this one?

Reply:
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/32652829/6c41e8d7-cec4-488f-9631-494a42bdead1)

Reply:
@virajsanghvi for denser display, I can remain this as extra small

Reply:
Yeah, I think this one is using it as custom form context, so think xs is the right choice here